### ISSUE TITLE
Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by melroy89** [ClassiCube/MCGalaxy#571]
Everything seems to work fine in Wine. Untill I press the Settings menu, it will crash and restarts.

Using Wine 5.0 (stable).

Logging:

```
----1/14/2021 5:14:47 PM ----
Type: IndexOutOfRangeException
Source: System.Windows.Forms
Message: Index was outside the bounds of the array.
Target: RefreshProperties
Trace:   at System.Windows.Forms.PropertyGrid.RefreshProperties (System.Boolean clearCached) [0x0001e] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.PropertyGrid.Refresh (System.Boolean clearCached) [0x00045] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.PropertyGrid.Refresh () [0x00013] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.PropertyGrid.OnFontChanged (System.EventArgs e) [0x00009] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Control.AssignParent (System.Windows.Forms.Control value) [0x000f0] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Control+ControlCollection.Add (System.Windows.Forms.Control value) [0x0011d] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.TabPage+TabPageControlCollection.Add (System.Windows.Forms.Control value) [0x00024] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at MCGalaxy.Gui.PropertyWindow.InitializeComponent () [0x0b95d] in <6b2d910a346d45dc99d7892444826fde>:0 
  at MCGalaxy.Gui.PropertyWindow..ctor () [0x0006c] in <6b2d910a346d45dc99d7892444826fde>:0 
  at (wrapper remoting-invoke-with-check) MCGalaxy.Gui.PropertyWindow..ctor()
  at MCGalaxy.Gui.Window.btnProperties_Click (System.Object sender, System.EventArgs e) [0x00007] in <6b2d910a346d45dc99d7892444826fde>:0 
  at System.Windows.Forms.Control.OnClick (System.EventArgs e) [0x0001f] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Button.OnClick (System.EventArgs e) [0x00037] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Button.OnMouseUp (System.Windows.Forms.MouseEventArgs mevent) [0x0009a] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Control.WmMouseUp (System.Windows.Forms.Messagem, System.Windows.Forms.MouseButtons button, System.Int32 clicks) [0x001c3] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Control.WndProc (System.Windows.Forms.Messagem) [0x005a0] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.ButtonBase.WndProc (System.Windows.Forms.Messagem) [0x0011f] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Button.WndProc (System.Windows.Forms.Messagem) [0x00056] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.OnMessage (System.Windows.Forms.Messagem) [0x00001] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Messagem) [0x000b3] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 
  at System.Windows.Forms.NativeWindow.Callback (System.IntPtr hWnd, System.Int32 msg, System.IntPtr wparam, System.IntPtr lparam) [0x00030] in <a1b7540aeb454df7a6cc546c7c9a66bb>:0 

-------------------------
```

Regards,

Melroy

**Comment by Vocaned** [comment]
Duplicate of #549, use mono if possible or configure the server through the files in the properties directory

**Comment by melroy89** [comment]
Alternatively use the CLI.exe instead.

**Comment by UnknownShadow200** [comment]
Should be fixed as of 2d87519af26089b770ece6b19d9eb475858a9ea0
